code_files/segment_tree_sum.cpp

```diff
@@ -0,0 +1,47 @@
+int tree[4 * N];
+
+int Get(int l, int r, int x, int lx, int rx) {
+    if (l > rx || r < lx) {
+        return 0;
+    }
+    if (l <= lx && rx <= r) {
+        return tree[x];
+    }
+    int m = (lx + rx) / 2;
+    int leftnode = Get(l, r, 2 * x, lx, m);
+    int rightnode = Get(l, r, 2 * x + 1, m + 1, rx);
+    return leftnode + rightnode;
+}
+
+int Get(int l, int r) {
+    return Get(l, r, 1, 1, n);
+}
+
+void Set(int pos, int v, int x, int l, int r) {
```

Comment:
change `int l, int r` to `int lx, int rx` so it matches with the `Get()` function